Makefile

```diff
@@ -437,6 +438,7 @@ tools: kustomize controller-gen operator-sdk
 
 .PHONY: install-tools
 install-tools: operator-sdk
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.2
```

Comment:
Does it make sense to install the linter maybe in our local `$(REPO)/bin` foulder? otherwise we overwrite the version installed in the `$GOPATH/bin`, right?

Reply:
I copied this from https://github.com/jaegertracing/jaeger/blob/main/Makefile#L392 ; assuming it was correct there.

Are you concerned about the Github builders, or are you concerned about people running `make install-tools` on their laptop?

Reply:
Discussed with @iblancasa - It seem we have several global software installations. We will address this later in a separate patch.